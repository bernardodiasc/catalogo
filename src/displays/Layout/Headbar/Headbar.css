.Layout-Headbar {
  width: 100%;
  height: 100%;
  border-bottom: 1px solid;
  display: flex;
  align-items: center;
}

.Layout-Headbar__brand {
  display: flex;
  background: silver;
  justify-content: center;
  align-items: center;
  width: 150px;
  height: 100%;
}

.Layout-Headbar__nav {
  display: flex;
  justify-content: flex-end;
  width: calc(100% - 150px);
}

.Layout-Headbar__nav li {
  display: inline-block;
  padding: 0 10px;
}
