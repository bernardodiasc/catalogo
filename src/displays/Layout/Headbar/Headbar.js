import React, { PureComponent } from 'react'
import './Headbar.css'

class Headbar extends PureComponent {
  static defaultProps = {}

  render() {
    return (
      <div className="Layout-Headbar">
        Brand
        <ul>
          <li><a>link</a></li>
          <li><a>link</a></li>
          <li><a>link</a></li>
        </ul>
      </div>
    )
  }
}

export default Headbar
