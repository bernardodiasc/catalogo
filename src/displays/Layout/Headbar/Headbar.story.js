import React from 'react'
import { storiesOf } from '@storybook/react'
import { withKnobs } from '@storybook/addon-knobs/react'
import StoryRouter from 'storybook-router'
import Docs from 'stories/containers/Docs'

import Headbar from './Headbar'
import HeadbarReadme from './README.md'
storiesOf('Displays/Layout/Headbar', module)
  .addDecorator(StoryRouter())
  .addDecorator(withKnobs)
  .add('Info', () => <Docs>{HeadbarReadme}</Docs>)
  .add('Example', () => <Headbar />)
  .add('Minimal render', () => <Headbar />)
