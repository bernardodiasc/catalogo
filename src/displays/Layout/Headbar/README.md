# Headbar

## How to use

```
import Headbar from 'displays/Layout/Headbar'
```

```
<Headbar />
```

## Properties

| propName | propType | defaultValue | isRequired |
| - | - | - | - |
