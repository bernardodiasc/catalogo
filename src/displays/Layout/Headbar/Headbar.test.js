import React from 'react'
import { shallow } from 'enzyme'

import Headbar from './Headbar.js'

it('renders without crashing', () => {
  const component = shallow(<Headbar />)
  expect(component).toMatchSnapshot()
})
