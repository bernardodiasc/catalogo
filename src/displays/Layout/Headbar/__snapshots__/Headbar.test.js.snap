// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="Layout-Headbar"
>
  <div
    className="Layout-Headbar__brand"
  >
    <Link
      replace={false}
      to="/"
    >
      Brand
    </Link>
  </div>
  <ul
    className="Layout-Headbar__nav"
  >
    <li>
      <Link
        replace={false}
        to="/quem-somos"
      >
        Quem Somos
      </Link>
    </li>
    <li>
      <Link
        replace={false}
        to="/anunciar"
      >
        Anunciar
      </Link>
    </li>
    <li>
      <Link
        replace={false}
        to="/franquia"
      >
        Franquia
      </Link>
    </li>
    <li>
      <Link
        replace={false}
        to="/contato"
      >
        Contato
      </Link>
    </li>
  </ul>
</div>
`;
